zwave_js/rootfs/etc/cont-init.d/config.sh

```diff
@@ -105,20 +105,22 @@ host_chassis=$(bashio::host.chassis)
 if [ "${host_chassis}" == "vm" ]; then
     soft_reset=false
     bashio::log.info "Virtual Machine detected, disabling soft-reset"
+elif bashio::config.true 'disable_soft_reset'; then
```

Comment:
Should we allow the user to override the automatic detection completely and enable soft-reset also on VM?

Reply:
I think so, yes. If they manage to configure the VM correctly, enabling soft-reset is good.

Reply:
OK so I have now changed it to a pick list of three options -> automatic, enabled, or disabled (defaults to automatic)